olm-catalog/serverless-operator/1.5.0/serverless-operator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -305,6 +305,8 @@ spec:
                       value: docker.io/maistra/proxyv2-ubi8:1.0.4
                     - name: IMAGE_3scale-kourier-control
                       value: registry.svc.ci.openshift.org/openshift/knative-v0.12.1:kourier
+                    - name: KOURIER_MANIFEST_PATH
+                      value: $KOURIER_MANIFEST_PATH
```

Comment:
I dunno if this works for us correctly. We'd need to reset this in productization I think? Can we instead set this to the "correct" path statically and replace it with `sed` for the CI case?

Reply:
Updated
Sure, we can use the path statically and `sed`. It would be more simple.